### PR TITLE
Add support to STBTLE-RF and STBTLE-1S module

### PR DESCRIPTION
**Pull request by cparata**
Hi Sandeep,
I would like to add the support to ST BLE modules (STBTLE-RF and STBTLE-1S) in the ArduinoBLE library. Basically these BLE modules work in coprocessor mode and they are connected to the main MCU via SPI interface. I successfully tested the ArduinoBLE library examples on X-NUCLEO-IDB05A1 shield (that mounts STMTLE-RF module) + a Nucleo board, on B-L475E-IOT01A (that mounts also STMTLE-RF module) and on STEVAL-MKSBOX1V1 (that mounts STMTLE-1S module) using the STM32duino environment. The patch, as you see, should not impact the Arduino boards because it is specific on ST boards, so it should be safe to integrate in the master repo.
Thanks in advance and Best Regards,
Carlo

### CODE REVIEW

**Comment by cparata** [comment]
Hi @sandeepmistry and @mbanzi ,
I understand your point of view and it is reasonable for me. We will create a fork of the ArduinoBLE library in the stm32duino repository. I guess that it will not be difficult to maintain and we can support directly people that want to use ArduinoBLE with ST platforms. I will submit in your official repository any PR that can improve the high level stack. Thanks for the explanation and Best Regards,
Carlo

**Comment by sandeepmistry** [comment]
Hi @cparata,

Thanks for your understanding, once your fork is public please link to it here for others to find in the future. Then I think we can go ahead and close this PR.

**Comment by cparata** [comment]
Hi @sandeepmistry ,
I generated the public fork at the following link:

https://github.com/stm32duino/STM32duinoBLE

Best Regards,
Carlo

File: README.md [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-
-For more information about this library please visit us at:
-https://www.arduino.cc/en/Reference/ArduinoBLE
+For more information about ArduinoBLE library please visit the official web page at:
+https://github.com/arduino-libraries/ArduinoBLE
 
 ## License
 
 ```
-Copyright (c) 2019 Arduino SA. All rights reserved.
```
Comment (fpistm) [inline README.md]: This should be kept.

Comment (cparata) [reply README.md]: OK, thanks.
Done!

**Comment by sandeepmistry** [comment]
Hi @cparata,

Thanks for the update and the link. For anyone looking to use this library with STM hardware in the future please visit: https://github.com/stm32duino/STM32duinoBLE

I'm going to close this pull request now as we discussed earlier in the thread.

**Pull request closed without merging**